spark-operator/src/main/java/org/apache/spark/k8s/operator/health/SentinelManager.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.k8s.operator.health;
```

Comment:
Although I understand this is about health checking, `health` sounds like a little ambiguous package name. Are we going to have other classes except `SentinelManager` class in this directory?

Reply:
As no additional health check is planned in near future - this is relocated into metrics.healthcheck to avoid confusion (As sentinel manager aims to publish metrics that reflects controller health)